viewer/api/factories.py

```diff
@@ -16,7 +16,7 @@ class Meta:
 
 
 class MetricFactory(factory.django.DjangoModelFactory):
-    name = factory.Sequence(lambda n: 'Metric %s' % n)
+    name = factory.Sequence(lambda n: 'scalar_cats_a_purring_%s' % n)
```

Comment:
It was a purr-fect time to make the factory metrics look more like real metrics.